.github/dependabot.yml

```diff
@@ -4,5 +4,3 @@ updates:
   directory: "/"
   schedule:
     interval: daily
-    time: "03:00"
-  open-pull-requests-limit: 10
```

Comment:
Good catch - looks like it was only applied through automation in 448868500934f923f4e93bc935366991e230b7e5.